### PR TITLE
docs: add example for awscc_ec2_flow_log

### PR DESCRIPTION
**Pull request by BondAnthony**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_ec2_flow_log/cloudwatch.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
+resource "awscc_ec2_flow_log" "example" {
+  deliver_logs_permission_arn = awscc_iam_role.example.arn
+  log_destination_type        = "cloud-watch-logs"
+  log_destination             = awscc_logs_log_group.example.arn
+  traffic_type                = "ALL"
+  resource_id                 = "vpc-07ddade55bee92f5f"
```
Comment (quixoticmonk) [inline examples/resources/awscc_ec2_flow_log/cloudwatch.tf:6]: @BondAnthony 
- Can we make the the `resource_id` a variable than the vpc_id present here ?
- Can we add tags for ec2_flow_log resource , as some in the schema has some differences for some resources ?

Comment (BondAnthony) [reply examples/resources/awscc_ec2_flow_log/cloudwatch.tf:6]: I've added tags to the example and updated the `resource_id` to use a variable.

File: examples/resources/awscc_ec2_flow_log/firehose.tf [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        Action = [
+          "s3:AbortMultipartUpload",
+          "s3:GetBucketLocation",
+          "s3:GetObject",
+          "s3:ListBucket",
+          "s3:ListBucketMultipartUploads",
+          "s3:PutObject"
+        ],
+        Resource = [
+          "${awscc_s3_bucket.fexample.arn}",
```
Comment (quixoticmonk) [inline examples/resources/awscc_ec2_flow_log/firehose.tf:66]: @BondAnthony  Should these be `awscc_s3_bucket.fexample.arn` and `awscc_kinesisfirehose_delivery_stream.example.arn` ?

Comment (BondAnthony) [reply examples/resources/awscc_ec2_flow_log/firehose.tf]: Thank you! This should have been `example` instead of `fexample`.

**Review by quixoticmonk — APPROVED** [review]

**Pull request merged**